support-frontend/assets/pages/contributions-landing/components/StripeCardForm/StripeCardForm.jsx

```diff
@@ -38,9 +40,30 @@ import { InlineError } from '@guardian/src-user-feedback';
 import { StripeCardFormField } from './StripeCardFormField';
 import './stripeCardForm.scss';
 import QuestionMarkHintIcon from 'components/svgs/questionMarkHintIcon';
+import { isValidZipCode } from 'helpers/formValidation';
 
 // ----- Types -----//
 
+type ZipCodeFieldStatus = "NONE" | "OPTIONAL" | "REQUIRED";
```

Comment:
A minor point, but `ZipCodeFieldStatus` suggests to me it's something about the state of the data in the field.
I'm trying to think of a more accurate name...`ZipCodeFieldMode`?

Reply:
Ahh yeah that's much better!